src/adhocracy_frontend/adhocracy_frontend/static/js/Packages/MyPackage/MyProposal/Module.ts

```diff
@@ -8,7 +9,11 @@ export var moduleName = "adhMyPackageMyProposal";
 export var register = (angular) => {
     angular
         .module(moduleName, [
+            AdhEmbedModule.moduleName,
             AdhHttpModule.moduleName,
         ])
+        .config(["adhEmbedProvider", (adhEmbedProvider : AdhEmbed.Provider) => {
+            adhEmbedProvider.registerDirective("my-proposal-detail");
```

Comment:
That's just the way `adhEmbedProvider` works. I do not say that it is good though.